src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1213,10 +1213,18 @@ def _need_update_container(self):
 
     def set_up_existing_container_update(self):
         if self.get_argument_force_single_container_updates():
+            if len(self.new_containerapp["properties"]["template"]["containers"]) == 0:
```

Comment:
Use `safe_get` to get value of `self.new_containerapp["properties"]["template"]["containers"])` and check `none` first.

Reply:
Updated